parser/src/tokenizer/token.rs

```diff
@@ -222,7 +223,7 @@ impl Token {
             Token::Input => sym::input,
             Token::Let => sym::Let,
             Token::Mut => sym::Mut,
-            Token::Ampersand => sym::Ampersand,
+            Token::Ampersand => sym::At,
```

Comment:
I'm confused by this: why is the ampersand token mapped to the at symbol?

Reply:
Just a typo whoops! will fix